concrete-commons/src/dispersion.rs

```diff
@@ -13,6 +13,7 @@
 //! which makes if possible to use any of those representations generically when noise must be
 //! defined.
 
+#[cfg(feature = "serde")]
```

Comment:
Here, I expected to see `#[cfg(feature="serde_serialize" ...)]` but I guess it also make sense that way.